api/sources/serializers.py

```diff
@@ -87,19 +87,19 @@ class Meta(UpdateSourceSerializer.Meta):
 
     def create(self, validated_data):
         icon_url = validated_data.pop('icon_url')
+        long_title = validated_data.pop('long_title')
+
         icon_file = self._fetch_icon_file(icon_url)
-        long_title = validated_data['long_title']
 
         label = long_title.replace(' ', '_').lower()
-
-        name = validated_data.get('name', label)
+        name = validated_data.pop('name', label)
 
         with transaction.atomic():
             source, created = models.Source.objects.get_or_create(
                 long_title=long_title,
                 defaults={
-                    'home_page': validated_data.get('home_page', None),
```

Comment:
Home page is a required field in the admin app. Would you mind making it not required so we don't have to add a random page if we edit the source? It would be nice if creating sources through the admin functioned the same way as through the API (i.e. not using home page).

Pretty sure it's this form https://github.com/CenterForOpenScience/SHARE/blob/72044b093af7c79f8844aa270395ffbac4557897/share/admin.py#L356

Reply:
Ohh, I did not know about that form. I'll update that to match the API's current behavior.